nsis-download/src/lib.rs

```
@@ -0,0 +1,67 @@
+#![allow(clippy::missing_safety_doc)]
```

lets add some safety docs instead of silencing the error, we should document what the NSIS stack should have and warn to not pop anything else other than the function input.